libr/core/cmd_print.c

```diff
@@ -3273,26 +3283,32 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 		}
 	}
 	switch (mode) {
-		case 'j':
-			pj_end (pj);
-			pj_end (pj);
-			r_cons_println (pj_string (pj));
-			pj_free (pj);
-			break;
-		case 'h': {
-			r_cons_printf ("\n%s\n", r_table_tofancystring (t));
-			r_table_free (t);
-			break;
+	case 'j':
+		pj_end (pj);
+		pj_end (pj);
+		r_cons_println (pj_string (pj));
+		break;
+	case 'h': {
+		char *table_string = r_table_tofancystring (t);
```

Comment:
maybe worth checking for null

Reply:
Makes sense, tofancystring sounds like nontrivial operation which might fail. It doesn't seem to have much error checking right now but might in future.